### ISSUE TITLE
sceen is not unlocked on incoming calls, notifications or when calling out

### ISSUE THREAD

**Issue by Danfro** [ubports/ubuntu-touch#536]
- Device: BQ E5 vegetahd
- Channel: RC (ubports OTA 4), same on ubports stable OTA3 and last canonical image
- Build: Ubuntu 15.04 (R15) - armhf 20171221

### bug description
For some while now I do experience the following behaviour:
a) On incoming calls the sound is played but the screen stays blank. The window where you can swipe to accept or refuse a call is not visible. Therefore I can not take calls easily. By pressing the power-button the screen becomes visible for a short time, long enough to swipe to take the call eventually. After taking the call the screen goes blank again.
b) On notifications/alarms i.e. from notes app or on incoming SMS the sound is played but the screen also stays blank.
c) When doing a call, after pressing the call-button the screen goes blank and stays that way. To end the call I have to press the power-button to turn on the screen and eventually manage to press the red call-button.
d) When plugging the phone into a usb-port on my laptop the screen stays blank.

Unfortunately that happens not always. Sometimes the phone works just fine. Would be to easy. :-)
I could not pin it down to some specific actions that would trigger a change in behaviour.
I was running the stable channel (OTA3) since last week and tried the rc channel to see if that would solve this issue.

### Expected behavior
a) On incoming call the window for taking or refusing calls should become visible
b) On notifications/SMS the screen should turn on an show a summary of the message.
c) On calling the screen should only turn blank when holding the phone close to my ear.
d) When plugging in the screen should turn on.

### Logfiles and additional information
[dmesg.txt](https://github.com/ubports/ubuntu-touch/files/1835253/dmesg.txt)
[logcat.txt](https://github.com/ubports/ubuntu-touch/files/1835254/logcat.txt)

With logviewer I did find an error message in
maliit-server, msync and sync-monitor:
"No such interface 'org.freedesktop.DBus.Properties' on object at path /"

**Comment by Danfro** [comment]
This problem is similar to #153 .
Just today I found out (before reading #153) that pressing the top left corner helps.
So it probably is a hardware issue. :-(

**Comment by Danfro** [comment]
It seems that several other users have the same problem. It does not seem to be likely that all of them do have hardware faults. Therefore I do reopen this issue because it is still valid (16.04 rc 2018-W44/2).

**Comment by Danfro** [comment]
#195

**Comment by Flohack74** [comment]
You are sure this is not a duplicate of #195 ?

**Comment by Danfro** [comment]
I can definitely confirm that this is not a hardware issue (@Flohack74, @Rom4nB,  @NeoTheThird) as suspected in #153.

I was having the issue as given above.
-> Installed UT Tweak Tool yesterday.
= The phone worked normally - no problems!
-> Today I did uninstall UT Tweak Tool.
= The problem was back again.
-> I did uninstall another app (Talaan).
= The phone works normally - no problems!
I left it here because that's so much easier to work with. ;-)

**!!! The apps itself do not cause the issue. I could have chosen any other app. I am giving the app-names only for whoever may have a look at the logs provided in case that helps!!!**
I did install several builds of notes-app for testing during the evening. But because the app was on the phone and stays on the phone it did not change the behaviour of my phone.

Here some logs:
[UTdmesg2018-11-12.txt](https://github.com/ubports/ubuntu-touch/files/2573783/UTdmesg2018-11-12.txt)
[UTlogcat2018-11-12.txt](https://github.com/ubports/ubuntu-touch/files/2573785/UTlogcat2018-11-12.txt)
